Tasks/KubernetesV0/src/kubernetes.ts

```diff
@@ -72,6 +72,15 @@ function executeKubectlCommand(clusterConnection: ClusterConnection) : any {
     var command = tl.getInput("command", true);
     var result = "";
     var ouputVariableName =  tl.getInput("kubectlOutput", false);  
+    var telemetry = {
+        registryType: registryType,
+        command: command
+    };
+
+    console.log("##vso[telemetry.publish area=%s;feature=%s]%s",
```

Comment:
This will log for only the commands specified in the Commands section. We have secrets and config maps section as well where we run the 'create' command if the user has specified a secret or a configmap name.